setup.cfg

```diff
@@ -1,6 +1,9 @@
+[bumpversion]
+current_version = 0.0.3
+
```

Comment:
why are you adding bumpversion to python cfg file ?
it does not belong here.
have you read how that bump ver action works? I think there should be a file specifically for that.

Reply:
Bump version automatically adds this to the setup.cfg file in staging. Then in the later steps staging merges back into dev and this gets added there. I think merging into staging without it might have caused an error. 

From what I've looked up, it doesn't matter that `[bumpversion]` has its own .cfg file as long as the location is specified in the workflow (which it is in this case). It does so in the following workflow step:

```
 uses: FragileTech/bump-version@main
      with:
        current_version: "${{ env.current_version }}"
        files: setup.cfg
```

Reply:
I looked at sdk repo. I think we are using `.bumpversion.cfg`. follow the same way.